src/main/java/org/opensearch/neuralsearch/query/NeuralSparseQueryBuilder.java

```diff
@@ -189,15 +191,19 @@ public NeuralSparseQueryBuilder getCopyNeuralSparseQueryBuilderForTwoPhase(float
             .neuralSparseQueryTwoPhaseInfo(
                 new NeuralSparseQueryTwoPhaseInfo(NeuralSparseQueryTwoPhaseInfo.TwoPhaseStatus.PHASE_TWO, pruneRatio, pruneType)
             );
+
+        // If the raw tokes are provided through the query directly then split it directly
```

Comment:
It a bit hard to get the idea of this comment, I put below my understanding, feel free to edit it
```suggestion
        // If raw tokens are provided directly in the query, split them without additional processing
```

Reply:
ok